js/join.js

```diff
@@ -182,18 +182,65 @@ function arePersonalDetailsValid() {
     country.value.trim() != ''
   );
 }
-
+console.log(introduction.value);
 function introPageValidator() {
+  let countIntroduction = 0;
+  let textAreaIntroduction = Array.from(
+    document.querySelector('#introduction').value,
+  );
+  for (let i = 0; i < textAreaIntroduction.length; i++) {
+    let char = textAreaIntroduction[i];
+    if (/S+/.test(char)) {
+      countIntroduction++;
+    }
+  }
+
+  let textAreaSkills = Array.from(document.querySelector('#skills').value);
+  let countSkills = 0;
+  for (let i = 0; i < textAreaSkills.length; i++) {
+    let char = textAreaSkills[i];
+    if (/S+/.test(char)) {
+      countSkills++;
+    }
+  }
+
+  let textAreaCollege = Array.from(document.querySelector('#college').value);
+  let countCollege = 0;
+  for (let i = 0; i < textAreaCollege.length; i++) {
+    let char = textAreaCollege[i];
+    if (/S+/.test(char)) {
+      countCollege++;
+    }
+  }
+
+  let textAreaForFun = Array.from(document.querySelector('#forFun').value);
+  let countForFun = 0;
+  for (let i = 0; i < textAreaForFun.length; i++) {
+    let char = textAreaForFun[i];
+    if (/S+/.test(char)) {
+      countForFun++;
+    }
+  }
+
+  let textAreafunFact = Array.from(document.querySelector('#funFact').value);
+  let countfunFact = 0;
+  for (let i = 0; i < textAreafunFact.length; i++) {
+    let char = textAreafunFact[i];
+    if (/S+/.test(char)) {
+      countfunFact++;
+    }
+  }
+
```

Comment:
This can't be refactor according to my perspective. Because when user filled correctly data, then the nextbutton will enable, before that this method checks the data validation